pilot/model/service.go

```diff
@@ -214,6 +215,7 @@ type ServiceInstance struct {
 	Labels           Labels          `json:"labels,omitempty"`
 	AvailabilityZone string          `json:"az,omitempty"`
 	ServiceAccount   string          `json:"serviceaccount,omitempty"`
+	ManagementPorts  PortList        `json:"managementports,omitempty"`
```

Comment:
Why is this coming into the service model? Also, thansk to kubernetes, an endpoint can be part of multiple service instances. if anything, this has to be in the endpoint.